docs/content/drafts/DropdownMenu2.mdx

```diff
@@ -0,0 +1,308 @@
+---
+component_id: dropdown_menu
+title: DropdownMenu v2
+status: Alpha
+source: https://github.com/primer/react/tree/main/src/DropdownMenu2.tsx
+storybook: '/react/storybook?path=/story/composite-components-dropdownmenu2'
+description: Use DropdownMenu to select a single option from a list of menu options.
+---
+
+import {Box, Text, Avatar} from '@primer/react'
+import {DropdownMenu, ActionList} from '@primer/react/drafts'
+import {Props} from '../../src/props'
+import State from '../../components/State'
+import {CalendarIcon, IterationsIcon, NumberIcon, SingleSelectIcon, TypographyIcon} from '@primer/octicons-react'
+
+<br />
+
+<State default={1}>
+  {([selectedIndex, setSelectedIndex]) => {
+    const fieldTypes = [
+      {icon: TypographyIcon, name: 'Text'},
+      {icon: NumberIcon, name: 'Number'},
+      {icon: CalendarIcon, name: 'Date'},
+      {icon: SingleSelectIcon, name: 'Single select'},
+      {icon: IterationsIcon, name: 'Iteration'}
+    ]
+    const selectedType = fieldTypes[selectedIndex]
+    return (
+      <Box sx={{border: '1px solid', borderColor: 'border.default', borderRadius: 2, padding: 6}}>
+        <Text sx={{fontSize: 1, mb: 1}}>Select field type:</Text>
```

Comment:
Visually this example feels very similar to form selects. Is this correct usage for this kind of component? Should we remove label here? 

To avoid confusion, shall we clarify that users use `Select` and `InputGroup` if they need labels? cc. @mperrotti 

![Screenshot 2022-01-14 at 14 43 52](https://user-images.githubusercontent.com/13340707/149534252-b7a46af0-a949-4ce6-9e46-85a813c51922.png)

Reply:
Oof, good point. I can remove the label here to avoid confusion